controllers/staff.js

```diff
@@ -1,6 +1,58 @@
 const db = require('../db');
 const {Roles} = require('../auth')
+const PS = require('pg-promise').PreparedStatement;
+
+const psGetStaff = new PS({ name: 'get-staff', text: 'SELECT * FROM Staff WHERE uid = $1;' });
+
+const psGetRestaurantId = new PS({ name: 'get-restaurant-id', text: 'SELECT rid FROM Staff WHERE uid = $1;' });
+
+const psGetTotalOrdersAndCost = new PS({ name: 'get-total-orders-and-cost', text:
+`
+WITH RestaurantOrderPair AS (
+    SELECT DISTINCT rid, oid
+    FROM Collates natural join Orders
+)
+SELECT count(distinct oid) as numOrders, sum(finalPrice) as totalCost, DATE_PART('month', orderTime) as mth, DATE_PART('year', orderTime) as yr
+FROM RestaurantOrderPair natural join Orders
+WHERE rid = $1 AND deliveredTime IS NOT NULL
+GROUP BY mth, yr;
+` });
+
+const psGetTop5Food = new PS({ name: 'get-top-5-food', text:
+`
+WITH FoodFreq AS (
+    SELECT fname, sum(qty) as totalQty, DATE_PART('month', orderTime) as mth, DATE_PART('year', orderTime) as yr
+    FROM Collates natural join Orders
+    WHERE rid = $1 
+    GROUP BY fname, mth, yr
+)
+SELECT fname, totalQty
+FROM FoodFreq
+WHERE mth = $2 AND yr = $3;
+ `});
```

Comment:
Are you trying to gather monthly data, or overall? plus i don't see any sorting logic here